packages/trader/src/sass/app/modules/smart-chart.scss

```diff
@@ -70,6 +70,40 @@
 
 /** @define chart-spot-label */
 .chart-spot-label {
+    &-profit {
+        position: relative;
+        top: -2.8rem;
```

Comment:
same here

Reply:
Same as above: Bottom is not the opposite of top in terms of positioning in css. If I use bottom 1.1rem, it will move it relatively to the bottom of the outer block, while I need to move it relatively to top of outer block.